rocAL/source/meta_data/text_file_meta_data_reader.cpp

```diff
@@ -67,17 +67,24 @@ void TextFileMetaDataReader::lookup(const std::vector<std::string> &image_names)
 }
 
 void TextFileMetaDataReader::read_all(const std::string &path) {
-    std::ifstream text_file(path.c_str());
+    std::ifstream text_file(_file_list_path.c_str());
     if (text_file.good()) {
         //_text_file.open(path.c_str(), std::ifstream::in);
         std::string line;
         while (std::getline(text_file, line)) {
-            std::istringstream line_ss(line);
-            int label;
-            std::string image_name;
-            if (!(line_ss >> image_name >> label))
-                continue;
-            add(image_name, label);
+            std::istringstream read_stream(line);
+            std::string file_name;
+            uint file_label;
+            if (!(read_stream >> file_name >> file_label)) {
+                break;
+            }
+            // process pair (file_name, label)
+            auto _last_id = file_name;
+            auto last_slash_idx = _last_id.find_last_of("\\/");
+            if (std::string::npos != last_slash_idx) {
+                _last_id.erase(0, last_slash_idx + 1);
```

Comment:
This erase is not done for the label_reader_folders I am curious how the file lookup works there